draft-bonaventure-mptcp-converters.mkd

```diff
@@ -431,13 +431,24 @@ Client                   Converter                       Server
 
 ## Sample Example of Incoming Converter-Assisted Multipath TCP Connection
 
-An example of an incoming converter-assisted Multipath TCP connection is depicted in {{fig-inestab}}. In order to support incoming connections from remote hosts, the client may use PCP {{RFC6887}} to instruct the converter to create dynamic mappings. Those mappings will be used by the converter to intercept an incoming TCP connection destined to the client and convert it into a Multipath TCP connection. 
+Remote Hosts can use the Transport Converter to connect to another Host.  We
```

Comment:
Please delete "Remote Hosts can use the Transport Converter to connect to another Host." because the sentence may be interpreted as if the remote host has a control on the Converter, which is not the case.